internal/server/mediaServer.go

```diff
@@ -129,11 +129,8 @@ func (m *MediaStorageServer) DownloadFileByName(
 	}
 
 	if file.FileName == "" {
-		stream.Send(&pbmedia.DownloadFileResponse{
-			Data: &pbmedia.DownloadFileResponse_Error{
-				Error: pbmedia.DownloadFileByNameError_FILE_NOT_FOUND,
-			},
-		})
+		stream.Send(nil) // returning nil response when the filename is left blank
```

Comment:
remove this please, the error is a sufficient response.

Reply:
Good catch! Done!